TOPIC8-MONGODB/src/main/java/data/CourseDao.java

```diff
@@ -0,0 +1,15 @@
+package data;
+
+import java.util.List;
+
+import models.Course;
+
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.data.mongodb.repository.Query;
+
+public interface CourseDao extends MongoRepository<Course, Long> {
+
+	@Query(value = "{ $and [ { teacherAssigned.idTeacher: ?0 }, { courseName :?0 }}")

```

Comment:
I tried to find a course relative to his teacher id and course name , but maybe its wrong